spec/requests/api/v1/drivers_spec.rb

```diff
@@ -6,7 +6,7 @@
   end
 
   it 'responds with drivers json' do
-    get '/api/v1/drivers'
+    response = doorkeeper_token.get('/api/v1/drivers')
```

Comment:
Would these tests make the lib/doorkeeper_spec tests redundant? I think we can remove those when the request tests and controller tests are completed.